Home_work_18/Task_2.py

```diff
@@ -0,0 +1,56 @@
+# Task 2nd
+
+class Boss:
+    def __init__(self, id_: int, name: str, company: str):
+        self.id = id_
+        self.name = name
+        self.company = company
+        self.__workers = []
+
+    def add_worker(self, worker):
+        if isinstance(worker, Worker):
+            self.__workers.append(worker)
+        else:
+            raise ValueError("The worker must be an instance of Worker class")
+
+
+class Worker:
+    def __init__(self, id_: int, name: str, company: str, boss: 'Boss'):
+        self.id = id_
+        self.name = name
+        self.company = company
+        self.__boss = boss
```

Comment:
I'd implement it as protected valiable (with single `_`):
```suggestion
        self._boss = boss
```

Reply:
Again, because of inheritance.